src/vivarium_gates_mncnh/components/pregnancy.py

```diff
@@ -95,13 +96,15 @@ def on_initialize_simulants(self, pop_data: SimulantData) -> None:
             pop_data
         )
         pregnancy_outcomes_and_durations[COLUMNS.CHILD_ALIVE] = "dead"
-        pregnancy_outcomes_and_durations[COLUMNS.CHILD_AGE] = np.nan
+        pregnancy_outcomes_and_durations[COLUMNS.CHILD_AGE] = CHILD_INITIALIZATION_AGE / 2
```

Comment:
We can't map an age of 0 so we just need it to be smaller than the midpoint of our early neonatal age group interval.